### PR TITLE
Check for CMake minimum version

### LINKED ISSUE #99: [conan center] check CMakeLists.txt file starts with cmake_minimum_required
if there is **CMakeLists.txt** file near to the **conanfile.py**, it should start from the `cmake_minimum_required` at very beginning
same for `test_package/CMakeLists.txt` (if exists)

### PR DESCRIPTION
**Pull request by uilianries**
- Try to find CMake files and validate if it contains cmake_minimum_required(VERSION ...)
- As some projects use CMAKE_SOURCE_DIR, the recipe need to replace the cmake file by another one (e.g. CMakeOriginalLists.cmake), that's why I'm checking pattern instead of "CMakeLists.txt" directly

closes #99 
Wiki: 

#### **<a name="KB-H028">#KB-H028</a>: "CMAKE MINIMUM VERSION"**

All CMake files added to recipe package should contain a minimal version (Not necessarily 2.8.11, it can be any version) available in the file:

```cmake
# CMakeLists.txt
cmake_minimum_required(VERSION 2.8.11)
project(conanwrapper)

...
```

This requirement is also valid for the `test_package`.

### CODE REVIEW

File: tests/test_hooks/conan-center/test_conan-center.py [hunk 1]
```diff
@@ -125,10 +128,44 @@ def test_conanfile_installer(self):
...
+            description = "whatever"
+            exports_sources = "CMakeLists.txt"
+
+            def package(self):
+                self.copy("CMakeLists.txt", dst="res")
+        """)
+        tools.save('conanfile.py', content=conanfile)
+        tools.save('CMakeLists.txt', content="project(foo CXX)")
+        output = self.conan(['create', '.', 'cmake/version@user/test'])
+        self.assertIn("ERROR: [CMAKE MINIMUM VERSION (KB-H028)] The CMake file 'CMakeLists.txt' " \
```
Comment (danimtb) [inline tests/test_hooks/conan-center/test_conan-center.py]: This message has to be improved as the user will not know if the cmakelist is the one coming from the recipe or from the test package

Comment (uilianries) [reply tests/test_hooks/conan-center/test_conan-center.py]: I see, I need to pass the cmakefile path. I gonna improve it.

Comment (uilianries) [reply tests/test_hooks/conan-center/test_conan-center.py]: @danimtb done

**Comment by uilianries** [comment]
#### **<a name="KB-H028">#KB-H028</a>: "CMAKE MINIMUM VERSION"**

All CMake files added to recipe package should contain a minimal version available in the file:

```cmake
# CMakeLists.txt
cmake_minimum_required(VERSION 2.8.11)
project(conanwrapper)

...
```

This requirement is also valid for the `test_package`.

**Comment by uilianries** [comment]
> LGTM. Maybe add the literal line to include, but I think everybody will understand the message

Sounds even better. okay, I'll update it

**Comment by uilianries** [comment]
all conflicts has been solved!

**Comment by danimtb** [comment]
Merging this after checking CI

**Comment by danimtb** [comment]
here is the link to the wiki entry: https://github.com/conan-io/conan-center-index/wiki/Error-Knowledge-Base#kb-h028-cmake-minimum-version

**Pull request merged**